iina/Logger.swift

```diff
@@ -34,11 +42,8 @@ class Logger {
     }
   }
 
-  var label: String
-  var level: LogLevel
-  var logToConsole = false
-
   static let enabled = Preference.bool(for: .enableLogging)
```

Comment:
I'm still not sold on this, versus a "disabled" log level. Could you explain this to me?

Reply:
Sure, if you have a good reason for it then go ahead and keep it.